### PR TITLE
es-1008 fix-orgId-discovery-settings

### PR DESCRIPTION
**Pull request by kenakaiser**

### CODE REVIEW

File: packages/veritone-react-common/src/components/ProfileMenu/InnerProfileMenu.js [hunk 1]
```diff
@@ -28,6 +28,9 @@ class InnerProfileMenu extends React.Component {
         lastName: string,
         image: string
       }),
+      organization: shape({
+        organizationId: number
+      }),
```
Comment (MmtBkn) [inline packages/veritone-react-common/src/components/ProfileMenu/InnerProfileMenu.js:33]: Remove unused prop

File: packages/veritone-react-common/src/components/ProfileMenu/index.js [hunk 2]
```diff
@@ -31,6 +31,9 @@ class ProfileMenu extends React.Component {
         firstName: string,
         lastName: string,
         image: string
+      }),
+      organization: shape({
+        organizationId: number
```
Comment (MmtBkn) [inline packages/veritone-react-common/src/components/ProfileMenu/index.js:36]: Remove unused prop

File: packages/veritone-react-common/src/components/ProfileMenu/InnerProfileMenu.js (hunk 1 above)
Comment (kenakaiser) [reply packages/veritone-react-common/src/components/ProfileMenu/InnerProfileMenu.js:33]: we are using here

File: packages/veritone-react-common/src/components/ProfileMenu/index.js (hunk 2 above)
Comment (kenakaiser) [reply packages/veritone-react-common/src/components/ProfileMenu/index.js:36]: i can remove here - but user is passed through

File: packages/veritone-react-common/src/components/ProfileMenu/InnerProfileMenu.js (hunk 1 above)
Comment (MmtBkn) [reply packages/veritone-react-common/src/components/ProfileMenu/InnerProfileMenu.js:33]: @kenakaiser I really don't see where can you point me the line that we are using?

Comment (kenakaiser) [reply packages/veritone-react-common/src/components/ProfileMenu/InnerProfileMenu.js:33]: line 60    
const orgId = get(this.props.user, 'organization.organizationId');

File: packages/veritone-react-common/src/components/ProfileMenu/InnerProfileMenu.js [hunk 3]
```diff
@@ -54,13 +57,12 @@ class InnerProfileMenu extends React.Component {
       'applicationId',
       'ea1d26ab-0d29-4e97-8ae7-d998a243374e'
     ]);
+    const orgId = get(this.props.user, 'organization.organizationId');
```
Comment (kenakaiser) [inline packages/veritone-react-common/src/components/ProfileMenu/InnerProfileMenu.js:60]: used here

File: packages/veritone-react-common/src/components/ProfileMenu/InnerProfileMenu.js (hunk 1 above)
Comment (MmtBkn) [reply packages/veritone-react-common/src/components/ProfileMenu/InnerProfileMenu.js:33]: Ohh... It's under the user object, got it.

**Pull request merged**